.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,17 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+name: Code Coverage Report
+
+on: [push, pull_request]
+
+jobs:
+  test:
```

Comment:
consider a more meaningful job name. Maybe `Report test coverage`

Reply:
Updated the name!